datahub-web-react/src/app/entity/dataset/profile/schema/components/SchemaHeader.tsx

```diff
@@ -213,12 +231,16 @@ export default function SchemaHeader({
                             </ValueButton>
                         </KeyValueButtonGroup>
                     )}
-                    {maxVersion > 0 &&
-                        (editMode ? (
-                            <ShowVersionButton onClick={() => setEditMode?.(false)}>Version Blame</ShowVersionButton>
-                        ) : (
-                            <ShowVersionButton onClick={() => setEditMode?.(true)}>Back</ShowVersionButton>
-                        ))}
+                    <ShowVersionButton onClick={() => setEditMode?.(false)}>Version Blame</ShowVersionButton>
```

Comment:
this diff intentional?

Reply:
no it's not! so sorry about that this is my bad - was just testing out how this looked with the button there - thanks for catching this